channels.yaml

```diff
@@ -238,3 +238,8 @@ releases:
     maxChannelServerVersion: v2.6.99
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
+  - version: v1.24.2+k3s2
+    minChannelServerVersion: v2.6.4-alpha1
```

Comment:
minChannelServerVersion should be updated to be `v2.6.7-alpha1`

Reply:
This is done.